boards: arm: stm32: Fix openocd warning

vslapik opened the pull request:
boards: arm: stm32: Fix openocd warning
    
    interface/stlink-v2-1.cfg and interface/stlink-v2.cfg are wrappers
    around interface/stlink.cfg, their inclusion trigger warning which
    this change addresses. Besides the warning there is nothing there
    except sourcing iterface/stlink.cfg directly.
    
    Signed-off-by: Vasili Slapik <vslapik@gmail.com>

    Warnings are below:
    "WARNING: interface/stlink-v2-1.cfg is deprecated, please switch to interface/stlink.cfg"
    "WARNING: interface/stlink-v2.cfg is deprecated, please switch to interface/stlink.cfg"

erwango reviewed (changes requested):
@vslapik , thanks for the change, please fix the commit messages issues (as reported by CI).
Otherwise looks fine.

vslapik committed:
boards: arm: stm32: Fix openocd warning

interface/stlink-v2-1.cfg and interface/stlink-v2.cfg are wrappers
around interface/stlink.cfg, their inclusion trigger warning which
this change addresses. Besides the warning there is nothing there
except sourcing iterface/stlink.cfg directly.

Signed-off-by: Vasili Slapik <vslapik@gmail.com>

vslapik commented:
@erwango , commit message is fixed.
